### DISCUSSION TITLE
Is it possible?: #_FILTER_YEARS shows a dropdown selection of the past years

### DISCUSSION THREAD

**Discussion by ebussinetz** [liedekef/events-made-easy#628] [Q&A]
On the Website of our choir, we have many past events, so it would be nice, if we could jump to the past years.
Thank you!

**Comment by liedekef** [comment]
Not right now, but the next version will have something like this. See this changeset:
https://github.com/liedekef/events-made-easy/commit/36d2cf298bb7270c33644f8a222c2df65b200c3e
